hazelcast-client/src/main/java/com/hazelcast/client/connection/nio/AbstractClientSelectionHandler.java

```diff
@@ -51,12 +51,8 @@ public final void onFailure(Throwable e) {
             sk.cancel();
         }
         connectionManager.destroyConnection(connection);
-        StringBuilder sb = new StringBuilder();
-        sb.append(Thread.currentThread().getName());
-        sb.append(" Closing socket to endpoint ");
-        sb.append(connection.getEndPoint());
-        sb.append(", Cause:").append(e);
-        logger.warning(sb.toString());
+        logger.warning(Thread.currentThread().getName() + " Closing socket to endpoint "
+                + connection.getEndPoint() + ", Cause:" + e);
```

Comment:
Shouldn't we use `logger.warning(String message, Throwable thrown)` here instead of adding the exception to the message? If you want to add we should have a space after the colon in `", Cause:"`